.github/workflows/liquid-repository-entity-cd-nuget-publish.yml

```diff
@@ -0,0 +1,87 @@
+        fi
+    
+    - name: (CI) Checkout repo on Pull Request
+      if: ${{ github.event_name == 'pull_request' }}
+      uses: actions/checkout@v2
+      with:
+        ref: ${{github.event.pull_request.head.ref}}
+        repository: ${{github.event.pull_request.head.repo.full_name}}
+        fetch-depth: 0 # required to eliminate shallow clone warning in Sonarcloud analysis
+        
+    - name: (CI/CD) Checkout repo
+      if: ${{ github.event_name != 'pull_request' }}
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0 # required to eliminate shallow clone warning in Sonarcloud analysis
+
+    # required by sonarcloud scanner
+    - name: (CI/CD) Setup Java JDK
+      uses: actions/setup-java@v1.4.3
+      with:
+        # The Java version to make available on the path. Takes a whole or semver Java version, or 1.x syntax (e.g. 1.8 => Java 8.x). Early access versions can be specified in the form of e.g. 14-ea, 14.0.0-ea, or 14.0.0-ea.28
+        java-version: 11.x
+        
+    - name: (CI/CD) Setup .NET Core SDK
+      uses: actions/setup-dotnet@v1.7.2
+      with:
+        # SDK version to use. Examples: 2.2.104, 3.1, 3.1.x
+        dotnet-version: 3.1.x
+
+    # required by sonarcloud scanner
+    - name: (CI/CD) Setup Sonar Scanner tool
+      run: dotnet tool install --global dotnet-sonarscanner
+
+    - name: (CI/CD) Install Test Reporting Tool
+      run: dotnet tool install --global dotnet-reportgenerator-globaltool
+
+    - name: (CI/CD) Restore dependencies
+      run: dotnet restore src/Liquid.Repository.EntityFramework/Liquid.Repository.EntityFramework.csproj
+
+
+
+    - name: (CI/CD) Build and Analyze Project
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
+        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN_EF }}
+      run: |
+        dotnet sonarscanner begin /k:"Avanade_Liquid.Repository.EntityFramework" /o:"avanade-1" /d:sonar.login="${{secrets.SONAR_TOKEN_EF}}" /d:sonar.host.url="https://sonarcloud.io" /d:sonar.cs.vstest.reportsPaths=$GITHUB_WORKSPACE/test/testresults/*.trx /d:sonar.coverageReportPaths=$GITHUB_WORKSPACE/test/coverlet/reports/SonarQube.xml
+        dotnet build src/Liquid.Repository.EntityFramework/Liquid.Repository.EntityFramework.csproj --configuration Release --no-restore
+        dotnet test test/Liquid.Repository.EntityFramework.Tests/*Tests.csproj --collect:"XPlat Code Coverage" --logger trx --results-directory $GITHUB_WORKSPACE/test/testresults
+        reportgenerator -reports:$GITHUB_WORKSPACE/test/testresults/**/coverage.cobertura.xml -targetdir:$GITHUB_WORKSPACE/test/coverlet/reports -reporttypes:"SonarQube"
+        dotnet sonarscanner end /d:sonar.login="${{secrets.SONAR_TOKEN_EF}}"
+    - name: (CD) Nuget Pack & Push to Nuget.org
+      if: ${{ github.event_name == 'push' }}
+      run: |
+        dotnet pack --no-build --configuration Release src/Liquid.Messaging.Azure/Liquid.Messaging.Azure.csproj --output .
```

Comment:
This line is incorrect, this must reference the src/Liquid.Repository.EntityFramework/Liquid.Repository.EntityFramework.csproj project